test_expect_failure/scalac_jvm_opts/BUILD

```diff
@@ -41,3 +45,17 @@ scala_library(
     # the `failing_scala_toolchain` is used.
     scalac_jvm_flags = ["-Xmx1G"],
 )
+
+proto_library(
```

Comment:
why is this in `expect_failure` We expect this to work don't we?

We should only put tests in here that won't build or won't pass with `bazel test //...`

This seems like it is a positive, not a negative test. It should go in `test/...`

Reply:
I can move it there. I noticed some tests that require extra arguments (eg, `bazel build --extra_toolchains ...`) were being put in `expect_failure` because targets in `test/` are all invoked with `bazel build //test/...` and there's no way to specify extra arguments.

Maybe we need another test directory called `test_manual` or something for this use case?

Reply:
yeah, I'd rather have a separate directory. This should be for tests or builds that we are expecting to fail. If we've accidentally allowed others to get in here, we should address that (but not necessarily in this PR).

Reply:
Sounds good. I've moved the passing manual tests to a `manual_test` directory.